components/maplibre/ban-map/index.js

```diff
@@ -298,10 +335,29 @@ function BanMap({map, isSourceLoaded, popup, address, setSources, setLayers, onS
     }
   }, [map, isSourceLoaded, address, setLayers])
 
+  useEffect(() => {
+    if (map && address?.type === 'numero' && positionsSource) {
+      positionsSource.setData({
+        type: 'FeatureCollection',
+        features: getPositionsFeatures(address)
+      })
+    }
```

Comment:
La state `positionsSource` sera recalculée dans tous cas, on ne fait que doubler le travail avec cette state. Simplifions en récupérer la source dynamique.
```suggestion
  useEffect(() => {
    const positionsSource = map.getSource('positions')
    if (address?.type === 'numero' && positionsSource) {
      positionsSource.setData({
        type: 'FeatureCollection',
        features: getPositionsFeatures(address)
      })
    }
  }, [map, address])
```